bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="doorbird" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="d101">
+		<label>Doorbird D101</label>
+		<description>Doorbird doorbell model D101</description>
+		<channels>
+			<channel id="doorbell" typeId="doorbell" />
+			<channel id="doorbellTimestamp" typeId="doorbellTimestamp" />
+			<channel id="doorbellImage" typeId="doorbellImage" />
+			<channel id="motion" typeId="motion" />
+			<channel id="motionTimestamp" typeId="motionTimestamp" />
+			<channel id="motionImage" typeId="motionImage" />
+			<channel id="light" typeId="light" />
+			<channel id="openDoor1" typeId="openDoor1" />
+			<channel id="getImage" typeId="getImage" />
+			<channel id="image" typeId="image" />
+			<channel id="imageTimestamp" typeId="imageTimestamp" />
+			<channel id="doorbellHistoryIndex" typeId="doorbellHistoryIndex" />
+			<channel id="doorbellHistoryTimestamp" typeId="doorbellHistoryTimestamp" />
+			<channel id="doorbellHistoryImage" typeId="doorbellHistoryImage" />
+			<channel id="motionHistoryIndex" typeId="motionHistoryIndex" />
+			<channel id="motionHistoryTimestamp" typeId="motionHistoryTimestamp" />
+			<channel id="motionHistoryImage" typeId="motionHistoryImage" />
+			<channel id="getDoorbellMontage" typeId="getDoorbellMontage" />
+			<channel id="doorbellMontage" typeId="doorbellMontage" />
+			<channel id="getMotionMontage" typeId="getMotionMontage" />
+			<channel id="motionMontage" typeId="motionMontage" />
+			<channel id="sipHangup" typeId="sipHangup" />
+			<channel id="restart" typeId="restart" />
+		</channels>
+		<config-description-ref uri="thing-type:doorbird:config" />
+	</thing-type>
+
+	<thing-type id="d210x">
+		<label>Doorbird D210x</label>
+		<description>Doorbird doorbell model D210x</description>
+		<channels>
+			<channel id="doorbell" typeId="doorbell" />
+			<channel id="doorbellTimestamp" typeId="doorbellTimestamp" />
+			<channel id="doorbellImage" typeId="doorbellImage" />
+			<channel id="motion" typeId="motion" />
```

Comment:
We can use system default channel type `trigger` for this channel.

Reply:
Outdated: see https://github.com/openhab/openhab2-addons/pull/5857/files#r331783379

Reply:
```suggestion
			<channel id="motion" typeId="system.motion" />
```

You can remove channel type `motion` ([line 82](https://github.com/openhab/openhab2-addons)) from this file./pull/5857/files#diff-f44bb87b35e8aa02edaf40d8ec8e6643R82